docs/data/sdks/typescript-react-native/migration.md

```diff
@@ -3,6 +3,322 @@ title: React Native SDK Migration Guide
+    ```
+
+### Tracking revenue
+
+#### `logRevenue()`
+
+The `logRevenue()` API maps to `revenue()`. `receipt` and `receiptSignature` is not supported.
+
+=== "@amplitude/react-native"
+
+    ```typescript
+    import { Amplitude } from '@amplitude/react-native';
+    
+    const userProperties = {
+        price: 3,
+        productId: 'com.company.productId',
+        quantity: 2,
+        revenueType: 'productRevenue',
+        eventProperties: {
+            key: 'value',
+        },
+    };
+
+    ampInstance.logRevenue(userProperties);
+    ```
+
+=== "@amplitude/analytics-react-native"
+
+    ```typescript
+    import { Revenue, revenue } from '@amplitude/analytics-react-native';
+
+    const event = new Revenue()
+        .setPrice(3)
+        .setProductId('com.company.productId')
+        .setQuantity(2)
+        .setRevenueType('productRevenue')
+        .setEventProperties({
+            key: 'value',
+        })
+
+    revenue(event);
+    ```
+You can also `setRevenue(6)` instead of `setPrice(3)` and `setQuantity(2)`.
+
+## Advanced topics
+
+### Identity
+
+#### Device ID
+
+As the maintenance React Native SDK is a wrapper of the maintenance iOS, maintenance Android SDK and maintenance Browser SDK and provides mappings from React Native to native SDK functions, device ID generation follows the native SDK of each platform. Learn more about device ID lifecycle of [maintenance iOS SDK](../../ios/#device-id-lifecycle) and [maintenance Android SDK](../../android/#device-id-lifecycle). You can also call `setAdvertisingIdForDeviceId()` or `setAppSetIdForDeviceId()` to set ADID or App Set ID as device ID.
+
+The new React Native SDK initializes the device ID in the following order, with the device ID being set to the first valid value encountered:
+
+1. Device ID of in the configuration on initialization
+2. "deviceId" value from URL param, for example http://example.com/?deviceId=123456789. If it runs on Web.
+3. Device ID in cookie storage.
+4. A randomly generated 36-character UUID
+
+#### Advertising IDs
+
+Maintenance React Native SDK supports setting an advertising ID as device ID by `setAdvertisingIdForDeviceId()` or `setAppSetIdForDeviceId()`. The new React Native SDK tracks ADID  by default as `config.trackingOptions.adid` defaults to `true`. However, the new React Native SDK doesn't support App Set ID, IDFA, or IDFV.
+
+### COPPA
+
+You can enable COPPA control by `enableCoppaControl()` in maintenance React Native SDK. The new React Native SDK doesn't support that API but you can still enable COPPA:
+
+* For iOS, IDFA and IDFV aren't tracked. For Android, you can turn off ADID by setting `config.trackingOptions.adid` to `false`
+* You can use an [enrichment Plugin](../#enrichment-type-plugin) to delete city in the payload.
+* You can turn off IP address tracking by setting `config.trackingOptions.ipAddress` to `false`
+* Location (latitude and longitude) isn't tracked
+
+### Session events
```

Comment:
I noticed while reviewing the TS RN SDK that session's are not tracked the same as they used to be in maintenance RN.

* RN legacy - used "mobile style" tracking on iOS and Android
* RN TS - uses "web" style session tracking on all platforms

I am going to put together a doc about improving session tracking in the TS SDK, but for now can you please call out this difference in the docs.

Reply:
Nice catch! I updated this section to https://pr-819.d19s7xzcva2mw3.amplifyapp.com/data/sdks/typescript-react-native/migration/#session-management and update the `setMinTimeBetweenSessionsMillis` in configuration to 





"similar to config.sessionTimeout, but they are not completely the same. Learn more about session management"

Reply:
Hi @Mercy811 I was wrong on this. Andrey pointed out that we are correctly tracking foreground/background in TS RN. See here: https://github.com/amplitude/Amplitude-TypeScript/pull/263

Reply:
Glad to hear that, then we don't need to fix it. Reverted previous change.